app/config/security.yml

```diff
@@ -60,16 +60,29 @@ nelmio_security:
             - '%saml_remote_idp_host%'
     csp:
         report_logger_service: monolog.logger.security
-        hosts:
-            - 'ajax.googleapis.com'
+        hosts: []
         content_types: []
         enforce:
-            report-uri: [/csp/report]
+            level1_fallback: false
+            browser_adaptive:
+                enabled: false
+            report-uri: "%router.request_context.base_url%/csp/report"
             default-src:
                 - 'self'
-                - 'data:'
             object-src:
                 - 'none'
+            style-src:
+                - 'self'
+                - 'netdna.bootstrapcdn.com'
```

Comment:
Is it acceptable to whitelist all those hosts?

Reply:
Our software depends on those dependencies, so yes, I think its acceptable. They are required by the translation bundle. So dropping those dependencies will require quite some work